### DISCUSSION TITLE
Do you have a roudmap to build Google Account SSO.

### DISCUSSION THREAD

**Discussion by RakuCloud** [Budibase/budibase#13937] [Ideas]
If NocoBase allows users to sign in and log in using their Google accounts (including enterprise accounts), it could become very convenient.

**Comment by ConorWebb96** [comment]
Hey @RakuCloud,

I think you may have accidentally posted to the wrong repository. Budibase, already has Google SSO.

**Reply by RakuCloud** [reply]
sorry！
